android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -712,9 +712,9 @@ public String getDeviceNameSync() {
   public String getSerialNumberSync() {
     try {
       if (Build.VERSION.SDK_INT >= 26) {
-        if (getReactApplicationContext().checkCallingOrSelfPermission(Manifest.permission.READ_PHONE_STATE) == PackageManager.PERMISSION_GRANTED) {
-          return Build.getSerial();
-        }
+        // There are a lot of conditions to access to getSerial api
+        // For details, see https://developer.android.com/reference/android/os/Build#getSerial()
+        return Build.getSerial();
```

Comment:
```suggestion
        // Rather than check each one, just try and rely on the catch below, for discussion on this approach, refer to
        // https://github.com/react-native-device-info/react-native-device-info/issues/1320
        return Build.getSerial();
```

Reply:
Unfortunately I can't just commit this and then merge it, I guess you have to click the commit button for it? I just wanted to link it to the discussion